### ISSUE TITLE
Adjust aria-label on Home page learn more links to start with "Learn more"

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-practices#2328]
> @a11ydoer @mcking65 8. The aria-label for all "learn more" links are missing a visible label name. Matt -I don't understand what is missing. You can't provide a visible label with aria-label. Please explain. Jemma - "Learn more" links looks good since all have good accessible names with aria-label. It does not need a visual label. Siri response- As per WCAG 2.5.3 "label in name" the aria-label must include a visible label along with the string. Ex "learn more about landmark guide" 9.

@shirsha thank you for the clarification. Now I understand what you mean. I think you mean to say that it is best practice in this case for the non-visual label to begin with the text of the visible label. I agree with that. Given timing, down to minutes now, we can follow up with that later.

_Originally posted by @mcking65 in https://github.com/w3c/aria-practices/issues/2323#issuecomment-1130299317_

**Comment by shirsha** [comment]
Thank you @mcking65  for raising this issue on my behalf

**Comment by aardrian** [comment]
Oops. I filtered for issues _since_ launch. It did not occur to me to look for an issue for a 2.5.3 violation filed _prior_ to launch. I guess I figured a WAI site would not launch with known WCAG violations.
